sdk/storage/azblob/zt_chunkwriting_test.go

```diff
@@ -195,6 +201,46 @@ func (s *azblobUnrecordedTestSuite) TestGetErr() {
 	}
 }
 
+// nolint
+func (s *azblobUnrecordedTestSuite) TestSlowDestCopyFrom() {
+	p, err := createSrcFile(_1MiB + 500*1024) //This should cause 2 reads
+	if err != nil {
+		panic(err)
+	}
+	defer func(name string) {
+		_ = os.Remove(name)
+	}(p)
+
+	from, err := os.Open(p)
+	if err != nil {
+		panic(err)
+	}
+	defer from.Close()
+
+	br := newFakeBlockWriter()
+	defer br.cleanup()
+
+	br.stageDelay = 200 * time.Millisecond
+	br.errOnBlock = 0
+
+	errs := make(chan error, 1)
```

Comment:
The goroutine here is a bit weird, but I wanted to listen for the success condition in a `select` with a context so any potential regressions cause a useful error rather than just hanging.